get hash precision should be limited to 12, not 64

edit: I can pr this myself, just thought it would be good to open an issue :)

the `geohash.EncodeWithPrecision` call at [internal/server/crud.go:217](https://github.com/tidwall/tile38/blob/f23b46cfc09fdf9f3dc88c1ca63fb83098657087/internal/server/crud.go#L217) should not allow `precision > 12`, because [vendor/github.com/mmcloughlin/geohash/geohash.go:36](https://github.com/tidwall/tile38/blob/f23b46cfc09fdf9f3dc88c1ca63fb83098657087/vendor/github.com/mmcloughlin/geohash/geohash.go#L36) performs a slice index by subtracting from 12, which underflows `uint`.

this is a fairly easy fix, just change 64 to 12 at [internal/server/crud.go:213](https://github.com/tidwall/tile38/blob/f23b46cfc09fdf9f3dc88c1ca63fb83098657087/internal/server/crud.go#L213)

<details>
<summary>server logs</summary>

```
$ docker run --name tile38 tile38/tile38

   _______ _______
  |       |       |
  |____   |   _   |   Tile38 1.21.1 (c1ba2d8) 64 bit (amd64/linux)
  |       |       |   Port: 9851, PID: 1
  |____   |   _   |
  |       |       |   tile38.com
  |_______|_______|

2020/08/11 21:02:54 [INFO] Server started, Tile38 version 1.21.1, git c1ba2d8
2020/08/11 21:02:54 [INFO] AOF loaded 0 commands: 0.00s, 0/s, 0 bytes/s
2020/08/11 21:02:54 [INFO] Ready to accept connections at [::]:9851
panic: runtime error: slice bounds out of range [18446744073709551615:12]

goroutine 14 [running]:
github.com/mmcloughlin/geohash.EncodeWithPrecision(0x0, 0x0, 0xd, 0x40, 0xd)
        /home/travis/gopath/src/github.com/tidwall/tile38/vendor/github.com/mmcloughlin/geohash/geohash.go:36 +0xd0
github.com/tidwall/tile38/internal/server.(*Server).cmdGet(0xc000286000, 0xc000080d70, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/tidwall/tile38/internal/server/crud.go:217 +0x2b51
github.com/tidwall/tile38/internal/server.(*Server).command(0xc000286000, 0xc000080d70, 0xc0003b8000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/tidwall/tile38/internal/server/server.go:1032 +0x1714
github.com/tidwall/tile38/internal/server.(*Server).handleInputCommand.func4(0xc000080d70, 0xc0000fef00, 0xc00010fa18, 0xc000286000, 0xc0003b8000, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/travis/gopath/src/github.com/tidwall/tile38/internal/server/server.go:890 +0x301
github.com/tidwall/tile38/internal/server.(*Server).handleInputCommand(0xc000286000, 0xc0003b8000, 0xc000080d70, 0x0, 0x0)
        /home/travis/gopath/src/github.com/tidwall/tile38/internal/server/server.go:891 +0xa04
github.com/tidwall/tile38/internal/server.(*Server).netServe.func1(0xc000037550, 0xc000286000, 0xedbce0, 0xc00000e300)
        /home/travis/gopath/src/github.com/tidwall/tile38/internal/server/server.go:417 +0x476
created by github.com/tidwall/tile38/internal/server.(*Server).netServe
        /home/travis/gopath/src/github.com/tidwall/tile38/internal/server/server.go:322 +0x213
```

</details>

<details>
<summary>client logs</summary>

```
$ docker container exec -it tile38 tile38-cli

127.0.0.1:9851> set fleet 1 string foo
{"ok":true,"elapsed":"50.756µs"}

127.0.0.1:9851> get fleet 1
{"ok":true,"object":"foo","elapsed":"21.855µs"}

127.0.0.1:9851> get fleet 1 hash 1
{"ok":true,"hash":"s","elapsed":"12.477µs"}

127.0.0.1:9851> get fleet 1 hash 11
{"ok":true,"hash":"s0000000000","elapsed":"11.469µs"}

127.0.0.1:9851> get fleet 1 hash 12
{"ok":true,"hash":"s00000000000","elapsed":"11.079µs"}

127.0.0.1:9851> get fleet 1 hash 13
read tcp 127.0.0.1:46790->127.0.0.1:9851: read: connection reset by peer
```

</details>

--- a/internal/server/crud.go
+++ b/internal/server/crud.go
@@ -210,7 +210,7 @@ func (server *Server) cmdGet(msg *Message) (resp.Value, error) {
 			buf.WriteString(`,"hash":`)
 		}
 		precision, err := strconv.ParseInt(sprecision, 10, 64)
-		if err != nil || precision < 1 || precision > 64 {
+		if err != nil || precision < 1 || precision > 12 {
 			return NOMessage, errInvalidArgument(sprecision)
 		}
 		center := o.Center()

